src/main/java/com/openshift/internal/restclient/model/BuildConfig.java

```diff
@@ -62,7 +64,7 @@
 	private static final String BUILD_CONFIG_IMAGECHANGE_IMAGE = "imageChange.image";
 	private static final String BUILD_CONFIG_IMAGECHANGE_NAME = "imageChange.from.name";
 	private static final String BUILD_CONFIG_IMAGECHANGE_TAG = "imageChange.tag";
-	private static final String SOURCE_STRATEGY =  "spec.strategy";
+	private static final String BUILD_STRATEGY =  "spec.strategy";
```

Comment:
even though I'd agree with you in saying that shorter is better I'd still use BUILDCONFIG_STRATEGY for the sake of consistency with the other constants that use the prefix BUILDCONFIG_ (or BUILD_CONFIG)